microsoftgraph/docs-conceptual/authentication-commands.md

```diff
@@ -39,11 +39,31 @@ There are three ways to allow delegated access using `Connect-MgGraph`:
     Connect-MgGraph -AccessToken $AccessToken
     ```
 
+#### Use delegated access using own App Id
+
+To follow the least privilege model for the applications in your Azure AD tenant, you might be concerned about consenting to many permissions scopes to the Microsoft Graph PowerShell app over time. To avoid this, you can register your own app for use with Microsoft Graph PowerShell. This allows you to have more granular control.
+
+Follow the steps below to set up your own app:
+
+1. Browse to **Entra** > **App registrations** [adappreg.cmd.ms](https://entra.microsoft.com/#view/Microsoft_AAD_RegisteredApps/ApplicationsListBlade/quickStartType~/null/sourceType/Microsoft_AAD_IAM/Microsoft_AAD_RegisteredApps/ApplicationsListBlade) > **New Registration**
```

Comment:
I would remove the reference to _adppreg.cmd.ms_

Reply:
> I would remove the reference to _adppreg.cmd.ms_

I think it would be good to provide a link to the app registration page for quick access. I will however change the link name. I also forgot that we should be referring to Entra yet in docs until sometime in August. I will edit that as well.